docs/whats_new/version_ongoing.rst

```diff
@@ -43,3 +43,9 @@ adapt the record auxiliary to fit with the brand new double threaded auxiliary i
 Acroname Auxiliary adaption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 adapt the acroname auxiliary to fit with the brand new double threaded auxiliary interface
+
+Tester Present Sender
+^^^^^^^^^^^^^^^^^^^^^
+Add a context manager, tester present sender, to keep a UDS session open for
+more than 5 seconds
```

Comment:
awesome would be a link to the new feature    uds_auxiliary.rst

Reply:
Done